app/models/event.rb

```diff
@@ -0,0 +1,22 @@
+class Event < ApplicationRecord
+  has_many :event_members
+  has_many :users, through: :event_members
+
+  def to_icon(event_number:)
```

Comment:
I decided against using some more heavyweight solutions like activerecord serialization because we'll want to customize every payload anyways.  so I figured it's nice to just be more explicit and define the full custom payload as opposed to having you look back at the schema to see what else is going into the payload